### DISCUSSION TITLE
Why "/run/user/1000/doc/d8e2de0b/local"?

### DISCUSSION THREAD

**Discussion by vgivanovic** [borgbase/vorta#1279] [General]
When I select `/usr/local/` as my source directory, Vorta insists on substituting `/run/user/1000/doc/d8e2de0b/local`. (`/usr/local/` is a as-far-as-I-can-tell normal directory on my laptop's root drive.) I have some questions:

- Is this a Vorta, Borg, Flatpak, or Linux artifact?
- Is the substituted name stable across boots?
- Why? `/usr/local/` seems to me to be a perfectly good, unique, stable name for a directory.

Thanks for any clarifications. (Appended are the processes that I think are borg/vorta relevant.)

```
vladimir    2781       1  0 00:43 ?        00:00:00 /usr/lib/systemd/systemd --user
vladimir   34891    3049  0 09:17 tty1     00:00:01 xfce4-appfinder --collapsed
vladimir   34922   34891  0 09:18 tty1     00:00:00 bwrap --args 39 vorta
vladimir   34925    2781  0 09:18 ?        00:00:00 /usr/lib/flatpak-session-helper
vladimir   34929    2781  0 09:18 ?        00:00:00 server --sh -n /run/user/1000/.flatpak-helper/pkcs11-flatpak-34925 -
vladimir   34932       1  0 09:18 tty1     00:00:00 bwrap --args 39 xdg-dbus-proxy --args=41
vladimir   34933   34932  0 09:18 tty1     00:00:00 xdg-dbus-proxy --args=41
vladimir   34938   34922  0 09:18 tty1     00:00:00 bwrap --args 39 vorta
vladimir   34939   34938  1 09:18 tty1     00:00:18 /usr/bin/python3 /app/bin/vorta
vladimir   46772   34939 79 09:39 ?        00:05:36 /usr/bin/python3 /app/bin/borg create --list --progress --info --log

```

**Comment by real-yfprojects** [comment]
Please open an new issue for this and fill in the complete issue template including logs! The output of `ls -l usr/local` might also be relevant.

**Reply by vgivanovic** [reply]
OK. (I posted in Discussion because I wasn't sure it merited becoming an issue.)

**Reply by real-yfprojects** [reply]
No problem but it definitely is an issue that needs debugging.
